Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+
+COPY <<EOF /etc/yum.repos.d/mongodb-org-x86_64-6.0.repo
+[mongodb-org-x86_64-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/6.0/x86_64/
+gpgcheck=1
+enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc
+EOF
+
+COPY <<EOF /etc/yum.repos.d/mongodb-org-aarch64-6.0.repo
+[mongodb-org-aarch64-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/6.0/aarch64/
+gpgcheck=1
+enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc
+EOF
+
+RUN microdnf install shadow-utils jq yum yum-utils &&\
+    yum -y update &&\
```

Comment:
Worth revisiting if all of this is needed and relevant.


We have a [base](https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?container-tabs=security) that should be security updated and now we also perform updates on our own that creates another layer. If we use cache that layer would not be updated unless something above changes.

Reply:
I think only yum is absolutely necessary, the rest can be removed.

With the updates do you mean yum -y update?